xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -858,7 +882,6 @@ This member is an explicit interface member implementation. It can be used only
         <param name="item">The group to remove.</param>
         <summary>Removes a <see cref="T:System.Text.RegularExpressions.Group" /> from the collection. Calling this method always throws <see cref="T:System.NotSupportedException" />.</summary>
         <returns>This method is not supported. No value is returned.</returns>
```

Comment:
Returns should also mention that the method will throw. There's no case where a value will ever be returned.
Also, make sure to preserve the tags. They are autogenerated. If the tag won't have documentation, it should have the placeholder text "To be added." which is equivalent to being empty.
```suggestion
        <returns>Calling this method always throws <see cref="T:System.NotSupportedException" />.</returns>
        <remarks>To be added.</remarks>
```